tests/test_io.py

```diff
@@ -90,3 +96,39 @@ def test_clipboard_iterator():
         assert_type(read_clipboard(iterator=False, chunksize=1), TextFileReader),
         TextFileReader,
     )
+
+
+def test_hdf():
+    with ensure_clean() as path:
+        check(assert_type(DF.to_hdf(path, "df"), None), type(None))
+        check(assert_type(read_hdf(path), Union[DataFrame, Series, Index]), DataFrame)
```

Comment:
Should we check `to_hdf()` for a `Series`?  Also, I don't think there is an `Index.to_hdf()`, so is `Index` a possible result of `read_hdf()`?

Reply:
I thought one could store indices but nope. Remvoed.